lettuce/simulation.py

```diff
@@ -85,6 +85,10 @@ def _report(self):
         for reporter in self.reporters:
             reporter(self.i, self.flow.units.convert_time_to_pu(self.i), self.f)
 
+    def save_reporters(self):
+        for reporter in self.reporters:
+            reporter.save()
+
```

Comment:
This will break the reporter interface. So far, each reporter only needs a `__call__`. This code here would require each reporter to also implement `.save`, which is not a good idea.